component/core/src/main/java/org/exoplatform/social/core/listeners/ActivityTagMetadataListener.java

```diff
@@ -75,6 +80,11 @@ private void updateActivityTags(ExoSocialActivity activity) {
     Identity audienceIdentity = activityManager.getActivityStreamOwnerIdentity(activity.getId());
     long audienceId = Long.parseLong(audienceIdentity.getId());
     String content = getActivityBody(activity);
+    try {
+      content = HTMLSanitizer.sanitize(content);
+    } catch (Exception e) {
+      LOG.error("Error while sanitizing activity content", e);
```

Comment:
```suggestion
      LOG.warn("Error while sanitizing activity content {}", content, e);
```
It's better to use LOG.warn level when the error is not blocker to the whole system and that it doesn't need attention from Ops.
BTW, it's better to display additional information concerning the error for better problem analysis

Reply:
Okay